packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -103,4 +144,298 @@ describe('EthereumChainAdapter', () => {
       )
     })
   })
+
+  describe('broadcastTransaction', () => {
```

Comment:
Note to self: Might want to also test the unhappy path (https://github.com/shapeshift/unchained/blob/dae102cd30a99751a529ade6ad9ea249fe52cb5c/node/coinstacks/ethereum/api/src/controller.ts#L271).